srtp/srtp.c

```diff
@@ -3081,6 +3098,53 @@ srtp_crypto_policy_set_aes_cm_256_null_auth (srtp_crypto_policy_t *p)
 }
 
 #ifdef OPENSSL
+void
+srtp_crypto_policy_set_aes_cm_192_hmac_sha1_80(srtp_crypto_policy_t *p) {
```

Comment:
Any reason this function isn't sitting near the other functions with similar names?